livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -55,14 +73,18 @@ def streaming_supported(self) -> bool:
 class SpeechStream(ABC):
     @abstractmethod
     def push_frame(self, frame: rtc.AudioFrame) -> None:
+        """
+        Push a frame to be recognized, it is recommended to push
+        frames as soon as they are available
+        """
         pass
 
     @abstractmethod
-    async def flush(self) -> None:
-        pass
-
-    @abstractmethod
-    async def aclose(self) -> None:
+    async def aclose(self, *, wait: bool = True) -> None:
```

Comment:
Yes, most of the time you can ignore this argument. 
It can become useful when you want to stop the connection and you don't care about the rest of the outputs